bundle/manifests/intel.com_notebooks.yaml

```diff
@@ -46,9 +46,11 @@ spec:
                 git_uri:
                   description: Git reposiory to be used for image building
                   type: string
+                  default: https://github.com/openvinotoolkit/openvino_notebooks
```

Comment:
when we specify the default, it will be always visible in the yaml file. I think it might be better to hide the fields for testing purposes